MethodLINQ/MethodLINQ/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MethodLINQ
+{
+    internal class Program
+    {
+        public static int[] SortArray(int[] arr)
+        {
+            return arr.Where(x => x > 0)
+                      .OrderByDescending(x => x.ToString().Length)
+                      .ThenBy(x => x)
+                      .ToArray();
+        }
```

Comment:
Не совсем соотвествует заданию

> Написать метод, который по заданному массиву целых чисел получает новый массив из **различных** положительных чисел исходного массива, отсортированный по количеству знаков в числе в порядке убывания. Числа, имеющие одинаковое количество знаков, должны идти в порядке возрастания.

```suggestion
        public static int[] SortArray(int[] arr)
        {
            return arr.Where(x => x > 0)
                      .Distinct()
                      .OrderByDescending(x => x.ToString().Length)
                      .ThenBy(x => x)
                      .ToArray();
        }
```